### ISSUE TITLE
Actions no longer same padded from part title vs. view title bar

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#236223]
Refs: https://github.com/microsoft/vscode/commit/276e24792198ecb7e83533d575568b66da722064

Changing the `padding` to `2px` just for the part titles, makes the actions be misaligned with other toolbars, such as the custom title. In this case notice how a secondary sidebar below the custom title aligns the actions differently:

![Image](https://github.com/user-attachments/assets/eab8e8ad-584b-43ca-aa8f-57cd1b88d633)

I think we need to be careful of spreading `margin` or `padding` changes into views only so that we can ensure other locations still look the same.

Why not have the `2px` change on the toolbar itself?

**Comment by bpasero** [comment]
@benibenj what is the discussion part here exactly?

**Comment by benibenj** [comment]
Title bar and secondary side bar actions were never aligned on Windows and Linux due to WCO, so I don't really see this as a bug, but as a feature request as it was never properly solved. Also, I think it is more important to align icons between view containers and views which is why I haven't reverted the change yet, but I might still end up doing so.

I'm still thinking about the approach to fix this:
- Changing the padding on all toolbars to 2px seems wrong only to fix view/sidebar alignment
- Revert and change view padding back to 3px. Someone set it to 2px because 3 seemed a bit far apart for view title which I kind of agree with

**Comment by bpasero** [comment]
> Title bar and secondary side bar actions were never aligned on Windows and Linux due to WCO, so I don't really see this as a bug, but as a feature request as it was never properly solved.

Yes, on Windows and Linux. But on macOS specifically I pushed changes end of last year to align the layout controls with the actions below, so I am passionate about this, since your change broke that work.

To be clear: I like that you fixed the issue with your change and would not suggest to revert it back. I am just saying that we can do more to keep UI consistency in other places, including the editor title area as captured in https://github.com/microsoft/vscode/issues/237458

**Comment by bpasero** [comment]
Going back to `3px` and addressing the `1px` gap for "..." in SCM view differently. Refs https://github.com/microsoft/vscode/pull/263972
